topology/probes/k8s/namespace.go

```diff
@@ -44,25 +44,32 @@ type namespaceProbe struct {
 }
 
 func newObjectIndexer(g *graph.Graph) *graph.MetadataIndexer {
+	ownedByNamespace := []string{
```

Comment:
You could simply do

```
ownedByNamespaceFilter := filters.NewOrFilter(
			filters.NewTermStringFilter("Type", "deployment"),	
			filters.NewTermStringFilter("Type", "daemonset"),	
			filters.NewTermStringFilter("Type", "ingress"),	
			filters.NewTermStringFilter("Type", "job"),	
			filters.NewTermStringFilter("Type", "pod"),	
			filters.NewTermStringFilter("Type", "persistentvolume"),	
			filters.NewTermStringFilter("Type", "persistentvolumeclaim"),	
			filters.NewTermStringFilter("Type", "networkpolicy"),	
			filters.NewTermStringFilter("Type", "replicaset"),	
			filters.NewTermStringFilter("Type", "replicationcontroller"),	
			filters.NewTermStringFilter("Type", "service"),	
			filters.NewTermStringFilter("Type", "statefulset"),	
		)

filter := filters.NewAndFilter(filter := filters.NewAndFilter(
 		filters.NewTermStringFilter("Manager", managerValue),
                ownedByNamespaceFilter)
```

This will keep it readable as you did introducing a variable while avoiding to add a loop + array allocation, etc

A comment would do the job too.